app/src/main/java/com/tistory/dividendcalendar/presentation/notice/model/NoticeResponse.kt

```diff
@@ -0,0 +1,20 @@
+package com.tistory.dividendcalendar.presentation.notice.model
+
+data class NoticeResponse(
+    val country: String = "",
+    val items: List<ItemResponse> = emptyList()
+) {
+    data class ItemResponse(
+        val id: Int = -1,
+        val title: String = "",
+        val description: String = "",
+        val date: String = ""
+    )
+}
+
+fun NoticeResponse.ItemResponse.mapToItem() = NoticeItem(
+    id = id,
+    title = title,
+    description = description,
+    updateDate = date
```

Comment:
```suggestion
    data class ItemResponse(
        val id: Int = -1,
        val title: String = "",
        val description: String = "",
        val date: String = ""
    ) {
          fun mapToItem() = return NoticeItem(
          id = id,
          title = title,
          description = description,
          updateDate = date
    }
```
데이터 클래스 안에 mapper가 같이 들어있을 때는
이런식으로 만들면 깔끔할 것 같아 제안드려봅니다 ~

Reply:
우왕! 이렇게 사용할 수도 있네요! 감사합니다!